src/encoded/docs/schema-changes.md

```diff
@@ -154,44 +154,104 @@ This new object is an array of example objects that can successfully POST agains
+
+* In some situations even if we are not making any changes that may involve modifying existing properties and an upgrader step is not technically required; but if these involve several additional properties being added resulting in substantial schema changes, it is would be a good idea to update the schema version. For example if we are including ten new properties independent of the one's that already existed.
+
+**Follow the steps as outlined below**
+
+1. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object and increment the schema version. Up until schema version 8 for library object, a submitter could specify only one fragmentation method. With the new change being implemented, we would like to allow a list of fragmentation methods to be specified. 
+
+        "fragmentation_methods": {
+            "title": "Fragmentation methods",
+            "type": "array",
+            "uniqueItems": true,
+            "description": "A list of nucleic acid fragmentation methods and restriction enzymes used in library preparation.",
+            "items":{
+                "title": "Fragmentation method",
+                "description": "A short description or reference of the nucleic acid fragmentation protocol used in library preparation.",
+                "type": "string",
+                "enum": [
+                    "chemical (DNaseI)",
+                    "chemical (DpnII restriction)",
+                    "chemical (generic)",
+                    "chemical (HindIII restriction)",
+                    "chemical (HindIII/DpnII restriction)",
+                    "chemical (Illumina TruSeq)",
+                    "chemical (MboI restriction)",
+                    "chemical (micrococcal nuclease)",
+                    "chemical (NcoI restriction)",
+                    "chemical (Nextera tagmentation)",
+                    "chemical (RNase III)",
+                    "chemical (Tn5 transposase)",
+                    "chemical (MseI restriction)",
+                    "chemical (CviAII restriction)",
+                    "chemical (Csp6I restriction)",
+                    "n/a",
+                    "none",
+                    "see document",
+                    "shearing (Covaris generic)",
+                    "shearing (Covaris LE Series)",
+                    "shearing (Covaris S2)",
+                    "shearing (generic)",
+                    "sonication (Bioruptor generic)",
+                    "sonication (Bioruptor Pico)",
+                    "sonication (Bioruptor Plus)",
+                    "sonication (Bioruptor Twin)",
+                    "sonication (Branson Sonifier 250)",
+                    "sonication (Branson Sonifier 450)",
+                    "sonication (generic microtip)",
+                    "sonication (generic)",
+                    "sonication (Sonics VCX130)"
+               ]
+
+            }
+        }
 
```

Comment:
please choose shorter code excerpt, listing 20 fragmentation methods is not helpful to anyone reading the code

Reply:
this example is not ideal, since it is unclear what was in the schema in the first place (fragmentation_method) - that is changing into fragmentation_methods (plural). Maybe you can go with a simple property name change?